src/extended_navigation_controller.js

```diff
@@ -301,11 +301,24 @@ export class ExtendedNavigationController extends NavigationController {
   }
 }
 
+function addAriaInfo(workspace, navController) {
+  const injectionDiv = workspace.getInjectionDiv();
+  injectionDiv.setAttribute('role', 'application');
+  injectionDiv.setAttribute('aria-label', 'Blockly workspace');
```

Comment:
We probably don't want to say Blockly here if this is user-facing. (Maybe "Code workspace" instead, or ask Josh what they call the code area in CSFirst) You don't need to change it now but maybe file an issue to fix it before releasing.

Reply:
Good point. Do you want that to be a bigger thing about how to customize details like that, or just a simple issue for now and we can return to it as we pile up more related configuration details?

Reply:
I think just a simple issue is good!